frontend/src/components/DocumentUnitReferenceInput.vue

```diff
@@ -86,6 +87,41 @@ watch(
 
 <template>
   <div class="flex flex-col gap-24">
+    <div v-if="lastSavedModelValue.isEmpty" class="flex items-center gap-16">
+      <div class="flex items-center">
+        <InputField
+          id="caselaw"
```

Comment:
@leonie-koch Should we give it a more precise name so that we make sure the id is unique? Like `caselaw-reference-type`